types/foundry/client/application/sidebar/sidebar-tab/chat-log.d.ts

```diff
@@ -139,6 +139,14 @@ declare class ChatLog<TChatMessage extends ChatMessage = ChatMessage> extends Si
         createOptions: ChatMessageModificationContext,
     ): Promise<void>;
 
+    /**
+     * Process messages which execute a macro.
+     * @param {string} command  The chat command typed.
+     * @param {RegExpMatchArray} match  The RegExp matches.
+     * @private
+     */
+    protected _processMacroCommand(command: string, match: RegExpMatchArray): unknown;
```

Comment:
```suggestion
    protected _processMacroCommand(command: string, match: RegExpMatchArray): Promise<void>;
```

Reply:
Moved to https://github.com/foundryvtt/pf2e/pull/12772 instead.